app/models/template.rb

```diff
@@ -1,5 +1,6 @@
 class Template < ActiveRecord::Base
   include Exportable
+  attr_accessor :modify_locked, :modify_default
```

Comment:
How about making this an `attr_reader` in order to prevent modifications outside of this class (using the ignore_locking/ignore_default helpers)?